### ISSUE TITLE
Windows install sometimes fails

### ISSUE THREAD

**Issue by Tonus1** [marcoancona/MineTime#136]
Hi,

Can'y have MineTine installed. Failed with this trace :

```
2019-02-12 18:01:11> Program: Starting Squirrel Updater: --install .
2019-02-12 18:01:11> Program: Starting install, writing to C:\Users\directeur\AppData\Local\SquirrelTemp
2019-02-12 18:01:12> Program: About to install to: C:\Users\directeur\AppData\Local\Vysor
2019-02-12 18:01:12> CheckForUpdateImpl: Couldn't write out staging user ID, this user probably shouldn't get beta anything: System.IO.DirectoryNotFoundException: Impossible de trouver une partie du chemin d'accès 'C:\Users\directeur\AppData\Local\Vysor\packages\.betaId'.
   à System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   à System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   à System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   à System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   à System.IO.File.WriteAllText(String path, String contents, Encoding encoding)
   à Squirrel.UpdateManager.CheckForUpdateImpl.getOrCreateStagedUserId()
2019-02-12 18:01:12> CheckForUpdateImpl: Failed to load local releases, starting from scratch: System.IO.DirectoryNotFoundException: Impossible de trouver une partie du chemin d'accès 'C:\Users\directeur\AppData\Local\Vysor\packages\RELEASES'.
   à System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   à System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   à Squirrel.Utility.LoadLocalReleases(String localReleaseFile)
   à Squirrel.UpdateManager.CheckForUpdateImpl.<CheckForUpdate>d__2.MoveNext()
2019-02-12 18:01:12> CheckForUpdateImpl: Reading RELEASES file from C:\Users\directeur\AppData\Local\SquirrelTemp
2019-02-12 18:01:12> CheckForUpdateImpl: First run or local directory is corrupt, starting from scratch
2019-02-12 18:01:13> ApplyReleasesImpl: Writing files to app directory: C:\Users\directeur\AppData\Local\Vysor\app-2.1.2
2019-02-12 18:01:22> LogHost: Rigging execution stub for Vysor_ExecutionStub.exe to C:\Users\directeur\AppData\Local\Vysor\Vysor.exe
2019-02-12 18:01:28> ApplyReleasesImpl: Squirrel Enabled Apps: [C:\Users\directeur\AppData\Local\Vysor\app-2.1.2\Vysor.exe]
2019-02-12 18:01:39> ApplyReleasesImpl: Starting fixPinnedExecutables
2019-02-12 18:01:39> ApplyReleasesImpl: Examining Pin: Firefox.lnk
2019-02-12 18:01:39> ApplyReleasesImpl: Examining Pin: Internet Explorer.lnk
2019-02-12 18:01:39> ApplyReleasesImpl: Examining Pin: Microsoft  Windows Fax and Scan.lnk
2019-02-12 18:01:39> ApplyReleasesImpl: Examining Pin: Microsoft Excel 2010.lnk
2019-02-12 18:01:39> ApplyReleasesImpl: Examining Pin: Microsoft Word.lnk
2019-02-12 18:01:39> ApplyReleasesImpl: Examining Pin: Windows Explorer.lnk
2019-02-12 18:01:39> ApplyReleasesImpl: Fixing up tray icons
2019-02-12 18:01:39> ApplyReleasesImpl: cleanDeadVersions: for version 2.1.2
2019-02-12 18:01:39> ApplyReleasesImpl: cleanDeadVersions: exclude folder app-2.1.2
2019-02-12 18:01:43> InstallHelperImpl: Couldn't write uninstall icon, don't care: System.Net.WebException: Le format du chemin d'accès donné n'est pas pris en charge. ---> System.Net.WebException: Le format du chemin d'accès donné n'est pas pris en charge. ---> System.NotSupportedException: Le format du chemin d'accès donné n'est pas pris en charge.
   à System.Security.Permissions.FileIOPermission.EmulateFileIOPermissionChecks(String fullPath)
   à System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   à System.Net.FileWebStream..ctor(FileWebRequest request, String path, FileMode mode, FileAccess access, FileShare sharing, Int32 length, Boolean async)
   à System.Net.FileWebResponse..ctor(FileWebRequest request, Uri uri, FileAccess access, Boolean asyncHint)
   --- Fin de la trace de la pile d'exception interne ---
   à System.Net.FileWebResponse..ctor(FileWebRequest request, Uri uri, FileAccess access, Boolean asyncHint)
   à System.Net.FileWebRequest.GetResponseCallback(Object state)
   --- Fin de la trace de la pile d'exception interne ---
   à System.Net.FileWebRequest.EndGetResponse(IAsyncResult asyncResult)
   à System.Net.WebClient.GetWebResponse(WebRequest request, IAsyncResult result)
   à System.Net.WebClient.DownloadBitsResponseCallback(IAsyncResult result)
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   à Squirrel.UpdateManager.InstallHelperImpl.<CreateUninstallerRegistryEntry>d__5.MoveNext()
2019-09-04 13:20:29> Program: Starting Squirrel Updater: --install .
2019-09-04 13:20:29> Program: Starting install, writing to C:\Users\directeur\AppData\Local\SquirrelTemp
2019-09-04 13:20:30> Program: About to install to: C:\Users\directeur\AppData\Local\MineTime
2019-09-04 13:20:31> CheckForUpdateImpl: Couldn't write out staging user ID, this user probably shouldn't get beta anything: System.IO.DirectoryNotFoundException: Impossible de trouver une partie du chemin d'accès 'C:\Users\directeur\AppData\Local\MineTime\packages\.betaId'.
   à System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   à System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   à System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   à System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   à System.IO.File.WriteAllText(String path, String contents, Encoding encoding)
   à Squirrel.UpdateManager.CheckForUpdateImpl.getOrCreateStagedUserId()
2019-09-04 13:20:31> CheckForUpdateImpl: Failed to load local releases, starting from scratch: System.IO.DirectoryNotFoundException: Impossible de trouver une partie du chemin d'accès 'C:\Users\directeur\AppData\Local\MineTime\packages\RELEASES'.
   à System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   à System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   à Squirrel.Utility.LoadLocalReleases(String localReleaseFile)
   à Squirrel.UpdateManager.CheckForUpdateImpl.<CheckForUpdate>d__2.MoveNext()
2019-09-04 13:20:31> CheckForUpdateImpl: Reading RELEASES file from C:\Users\directeur\AppData\Local\SquirrelTemp
2019-09-04 13:20:31> CheckForUpdateImpl: First run or local directory is corrupt, starting from scratch
2019-09-04 13:20:33> ApplyReleasesImpl: Writing files to app directory: C:\Users\directeur\AppData\Local\MineTime\app-1.6.1
2019-09-04 13:20:38> LogHost: Rigging execution stub for MineTime_ExecutionStub.exe to C:\Users\directeur\AppData\Local\MineTime\MineTime.exe
2019-09-04 13:21:02> ApplyReleasesImpl: Squirrel Enabled Apps: [C:\Users\directeur\AppData\Local\MineTime\app-1.6.1\MineTime.exe]
2019-09-04 13:21:03> ApplyReleasesImpl: Couldn't run Squirrel hook, continuing: C:\Users\directeur\AppData\Local\MineTime\app-1.6.1\MineTime.exe: System.ComponentModel.Win32Exception (0x80004005): L'exécutable spécifié n'est pas une application valide pour cette plateforme de système d'exploitation.
   à System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   à System.Diagnostics.Process.Start()
   à System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   à Squirrel.Utility.<InvokeProcessAsync>d__11.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<>c__DisplayClass10_0.<<invokePostInstall>b__0>d.MoveNext()
2019-09-04 13:21:03> IEnableLogger: Failed to invoke post-install: System.ComponentModel.Win32Exception (0x80004005): L'exécutable spécifié n'est pas une application valide pour cette plateforme de système d'exploitation.
   à System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   à System.Diagnostics.Process.Start()
   à System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<>c.<invokePostInstall>b__10_6(ProcessStartInfo info)
   à Squirrel.EnumerableExtensions.ForEach[TSource](IEnumerable`1 source, Action`1 onNext)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<invokePostInstall>d__10.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Squirrel.Utility.<LogIfThrows>d__42.MoveNext()
2019-09-04 13:21:03> Unhandled exception: System.AggregateException: Une ou plusieurs erreurs se sont produites. ---> System.ComponentModel.Win32Exception: L'exécutable spécifié n'est pas une application valide pour cette plateforme de système d'exploitation.
   à System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   à System.Diagnostics.Process.Start()
   à System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<>c.<invokePostInstall>b__10_6(ProcessStartInfo info)
   à Squirrel.EnumerableExtensions.ForEach[TSource](IEnumerable`1 source, Action`1 onNext)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<invokePostInstall>d__10.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Squirrel.Utility.<LogIfThrows>d__42.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<ApplyReleases>d__2.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Squirrel.UpdateManager.<FullInstall>d__10.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   à Squirrel.Update.Program.<Install>d__4.MoveNext()
   --- Fin de la trace de la pile d'exception interne ---
   à System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   à System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   à System.Threading.Tasks.Task.Wait()
   à Squirrel.Update.Program.executeCommandLine(String[] args)
   à Squirrel.Update.Program.main(String[] args)
---> (Exception interne #0) System.ComponentModel.Win32Exception (0x80004005): L'exécutable spécifié n'est pas une application valide pour cette plateforme de système d'exploitation.
   à System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   à System.Diagnostics.Process.Start()
   à System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<>c.<invokePostInstall>b__10_6(ProcessStartInfo info)
   à Squirrel.EnumerableExtensions.ForEach[TSource](IEnumerable`1 source, Action`1 onNext)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<invokePostInstall>d__10.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Squirrel.Utility.<LogIfThrows>d__42.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<ApplyReleases>d__2.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Squirrel.UpdateManager.<FullInstall>d__10.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   à Squirrel.Update.Program.<Install>d__4.MoveNext()<---

2019-09-04 13:36:07> Program: Starting Squirrel Updater: --install . --rerunningWithoutUAC
2019-09-04 13:36:07> Program: Starting install, writing to C:\Users\directeur\AppData\Local\SquirrelTemp
2019-09-04 13:36:07> Program: About to install to: C:\Users\directeur\AppData\Local\MineTime
2019-09-04 13:36:07> Program: Install path C:\Users\directeur\AppData\Local\MineTime already exists, burning it to the ground
2019-09-04 13:36:08> CheckForUpdateImpl: Couldn't write out staging user ID, this user probably shouldn't get beta anything: System.IO.DirectoryNotFoundException: Impossible de trouver une partie du chemin d'accès 'C:\Users\directeur\AppData\Local\MineTime\packages\.betaId'.
   à System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   à System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   à System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   à System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   à System.IO.File.WriteAllText(String path, String contents, Encoding encoding)
   à Squirrel.UpdateManager.CheckForUpdateImpl.getOrCreateStagedUserId()
2019-09-04 13:36:08> CheckForUpdateImpl: Failed to load local releases, starting from scratch: System.IO.DirectoryNotFoundException: Impossible de trouver une partie du chemin d'accès 'C:\Users\directeur\AppData\Local\MineTime\packages\RELEASES'.
   à System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   à System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   à Squirrel.Utility.LoadLocalReleases(String localReleaseFile)
   à Squirrel.UpdateManager.CheckForUpdateImpl.<CheckForUpdate>d__2.MoveNext()
2019-09-04 13:36:08> CheckForUpdateImpl: Reading RELEASES file from C:\Users\directeur\AppData\Local\SquirrelTemp
2019-09-04 13:36:08> CheckForUpdateImpl: First run or local directory is corrupt, starting from scratch
2019-09-04 13:36:09> ApplyReleasesImpl: Writing files to app directory: C:\Users\directeur\AppData\Local\MineTime\app-1.6.1
2019-09-04 13:36:13> LogHost: Rigging execution stub for MineTime_ExecutionStub.exe to C:\Users\directeur\AppData\Local\MineTime\MineTime.exe
2019-09-04 13:36:27> ApplyReleasesImpl: Squirrel Enabled Apps: [C:\Users\directeur\AppData\Local\MineTime\app-1.6.1\MineTime.exe]
2019-09-04 13:36:27> ApplyReleasesImpl: Couldn't run Squirrel hook, continuing: C:\Users\directeur\AppData\Local\MineTime\app-1.6.1\MineTime.exe: System.ComponentModel.Win32Exception (0x80004005): L'exécutable spécifié n'est pas une application valide pour cette plateforme de système d'exploitation.
   à System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   à System.Diagnostics.Process.Start()
   à System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   à Squirrel.Utility.<InvokeProcessAsync>d__11.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<>c__DisplayClass10_0.<<invokePostInstall>b__0>d.MoveNext()
2019-09-04 13:36:27> IEnableLogger: Failed to invoke post-install: System.ComponentModel.Win32Exception (0x80004005): L'exécutable spécifié n'est pas une application valide pour cette plateforme de système d'exploitation.
   à System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   à System.Diagnostics.Process.Start()
   à System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<>c.<invokePostInstall>b__10_6(ProcessStartInfo info)
   à Squirrel.EnumerableExtensions.ForEach[TSource](IEnumerable`1 source, Action`1 onNext)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<invokePostInstall>d__10.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Squirrel.Utility.<LogIfThrows>d__42.MoveNext()
2019-09-04 13:36:27> Unhandled exception: System.AggregateException: Une ou plusieurs erreurs se sont produites. ---> System.ComponentModel.Win32Exception: L'exécutable spécifié n'est pas une application valide pour cette plateforme de système d'exploitation.
   à System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   à System.Diagnostics.Process.Start()
   à System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<>c.<invokePostInstall>b__10_6(ProcessStartInfo info)
   à Squirrel.EnumerableExtensions.ForEach[TSource](IEnumerable`1 source, Action`1 onNext)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<invokePostInstall>d__10.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Squirrel.Utility.<LogIfThrows>d__42.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<ApplyReleases>d__2.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Squirrel.UpdateManager.<FullInstall>d__10.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   à Squirrel.Update.Program.<Install>d__4.MoveNext()
   --- Fin de la trace de la pile d'exception interne ---
   à System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   à System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   à System.Threading.Tasks.Task.Wait()
   à Squirrel.Update.Program.executeCommandLine(String[] args)
   à Squirrel.Update.Program.main(String[] args)
---> (Exception interne #0) System.ComponentModel.Win32Exception (0x80004005): L'exécutable spécifié n'est pas une application valide pour cette plateforme de système d'exploitation.
   à System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   à System.Diagnostics.Process.Start()
   à System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<>c.<invokePostInstall>b__10_6(ProcessStartInfo info)
   à Squirrel.EnumerableExtensions.ForEach[TSource](IEnumerable`1 source, Action`1 onNext)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<invokePostInstall>d__10.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Squirrel.Utility.<LogIfThrows>d__42.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   à Squirrel.UpdateManager.ApplyReleasesImpl.<ApplyReleases>d__2.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Squirrel.UpdateManager.<FullInstall>d__10.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   à Squirrel.Update.Program.<Install>d__4.MoveNext()<---
```

**Comment by marcoancona** [comment]
It seems the installer cannot access the installation location. According to https://github.com/Squirrel/Squirrel.Windows/issues/698 this is a common issue with Antivirus being too restrictive. We are investigating this.

**Comment by Tonus1** [comment]
Thanks for the light. I will try to get rid of this on my side as well

**Comment by marcoancona** [comment]
Did you manage to get MineTime installed in the meanwhile?

**Comment by LowLyan** [comment]
Hi!

I'm experiencing the same issue.
Same log...

The problem seems to be that the installer tries to create a .betaId folder.
It seems a unix/linux/Mac hidden folder, that in Windows cannot be implemented (Win
recognizes it as a file extension without a name!)

Let me know, I really wanna try the software.

Thanks

**Comment by marcoancona** [comment]
Just released 1.6.3. Does the problem persist?

**Comment by marcoancona** [comment]
Candidate solution: we have released a Beta version 1.6.5 with a new installer. If you face issues installing on Windows, please try [this release](https://github.com/marcoancona/MineTimePreview/releases/tag/v1.6.5) from our Preview repository.

**Technical explanation**
MineTime currently uses Electron built-in auto-update system, which is robust but only supports Squirrel.Windows as an installer. Unfortunately, Squirrel.Windows is buggy and not maintained anymore. This is a [known issue](https://github.com/electron-userland/electron-builder/issues/1171). The solution is to migrate to NSIS as Windows installer. However, NSIS is not supported by electron auto-update and therefore we will need to switch to [electron-updater](https://www.npmjs.com/package/electron-updater) too. We have integrated NSIS and electron-updater in a [beta release](https://github.com/marcoancona/MineTimePreview/releases/tag/v1.6.5) but we need to find a solution for a smooth migration of existing users before releasing.

**Comment by marcoancona** [comment]
NSIS is now used since 1.7.0. This should solve many issues on Windows. I will close this, feel free to comment or open a new issue if the problem persists.
